app/frontend/components/topNavigation/topNavigation.tsx

```diff
@@ -15,7 +15,7 @@ export const TopNavigation = (): JSX.Element => {
       <TopNavigationLink href="/time">Time</TopNavigationLink>
       {teamSlug && <TopNavigationLink href={`/${teamSlug}/time`}>Time</TopNavigationLink>}
       {teamSlug && <TopNavigationLink href={`/${teamSlug}/projects`}>Projects</TopNavigationLink>}
-      <TopNavigationLink href="/reports">Reports</TopNavigationLink>
+      {teamSlug && <TopNavigationLink href={teamSlug ? `/${teamSlug}/reports` : '/reports'}>Reports</TopNavigationLink>}
```

Comment:
We already check beforehand if there is a team slug. This if-condition is therefore unnecessary.
```suggestion
      {teamSlug && <TopNavigationLink href={`/${teamSlug}/reports`}>Reports</TopNavigationLink>}
```

Reply:
That, I really didn't notice 😊!